docs/index.html

```diff
@@ -99,7 +99,11 @@ <h2 class="display-5">Joshua Zenn</h2>
             <div class="bg-light me-md-1 pt-1 px-1 pt-md-1 px-md-1 text-center text-black overflow-hidden">
                 <div class="my-3 py-3">
                     <h2 class="display-5">James Pruitt</h2>
-                    <p class="lead">And an even wittier subheading.</p>
+                    <p class="lead">Frontend/Backend Developer [Records Page].</p>
+                    <div class="d-md-flex flex-md-equal">
+                        <a href="https://github.com/spaceCowboy2071"><i class="bi bi-github"></i></a>
+                        <a href="https://www.linkedin.com/in/jsp-codes/"><i class="bi bi-linkedin"></i></a>
+                    </div>
                 </div>
                 <img class="bg-light shadow-sm mx-auto" style="width: 80%; height: 300px; border-radius: 21px 21px 0 0;" src="assets/img/feature1.png" />
```

Comment:
If you wanted to change the profile picture that appears, you can change the `src` attribute here. Otherwise if you don't want to include a headshot you can delete this element since it's just a placeholder.